use screenshot sorce error

### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.


### YOLOv8 Component

Detection

### Bug

 File "D:\2_yolov8\python_www\venv\lib\site-packages\ultralytics\yolo\engine\predictor.py", line 248, in stream_inference
    im = self.preprocess(im0s)
  File "D:\2_yolov8\python_www\venv\lib\site-packages\ultralytics\yolo\engine\predictor.py", line 123, in preprocess
    im = np.stack(self.pre_transform(im))
  File "D:\2_yolov8\python_www\venv\lib\site-packages\ultralytics\yolo\engine\predictor.py", line 144, in pre_transform
    return [LetterBox(self.imgsz, auto=auto, stride=self.model.stride)(image=x) for x in im]
  File "D:\2_yolov8\python_www\venv\lib\site-packages\ultralytics\yolo\engine\predictor.py", line 144, in <listcomp>
    return [LetterBox(self.imgsz, auto=auto, stride=self.model.stride)(image=x) for x in im]
  File "D:\2_yolov8\python_www\venv\lib\site-packages\ultralytics\yolo\data\augment.py", line 581, in __call__
    img = cv2.resize(img, new_unpad, interpolation=cv2.INTER_LINEAR)
cv2.error: OpenCV(4.7.0) D:\a\opencv-python\opencv-python\opencv\modules\imgproc\src\resize.cpp:4065: error: (-215:Assertion failed) inv_scale_x > 0 in function 'cv::resize'

### Environment

_No response_

### Minimal Reproducible Example

_No response_

### Additional

_No response_

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

--- a/ultralytics/data/loaders.py
+++ b/ultralytics/data/loaders.py
@@ -156,7 +156,7 @@ class LoadScreenshots:
         s = f'screen {self.screen} (LTWH): {self.left},{self.top},{self.width},{self.height}: '
 
         self.frame += 1
-        return str(self.screen), im0, None, s  # screen, img, original img, im0s, s
+        return [str(self.screen)], [im0], None, s  # screen, img, vid_cap, string
 
 
 class LoadImages:

